ChainSafe/web3.unity
# Event subscriptions via tx receipt

Event subscriptions via tx receipt

Button added to evm area in the sample scene

Docs updated here: https://github.com/ChainSafe/game-docs-v2/pull/108

To test:
Login with a wallet on sepolia and press the event data via tx button in the sample scene

Expected result:
The wallet and amount of increase is retrieved from the contract event and sent to the console
